packages/react-dom/src/__tests__/ReactBrowserEventEmitter-test.js

```diff
@@ -146,13 +129,12 @@ describe('ReactBrowserEventEmitter', () => {
    * to instead.
    */
   it('should not invoke handlers if ReactBrowserEventEmitter is disabled', () => {
-    const inst = createInitialInstance();
+    createInitialInstance();
     ReactBrowserEventEmitter.setEnabled(false);
-    const child = getChildInstance(inst);
-    ReactTestUtils.SimulateNative.click(ReactDOM.findDOMNode(child));
+    ReactTestUtils.SimulateNative.click(ReactDOM.findDOMNode(CHILD));
```

Comment:
Suggestion: let's remove any use of `Simulate` and `SimulateNative` in this file.

Instead, make sure that we add `container` to `document.body` in `beforeEach`, remove it from `document.body` in `afterEach`, and then use browser native dispatch:

```js
node.dispatchEvent(
  new MouseEvent('click', {bubbles: true, cancelable: true}),
);
```